### PR TITLE
Setting to enable/disable full screen navigation swipe gesture

### PR DESCRIPTION
**Pull request by hjiangsu**
## Pull Request Description

Adds a setting to enable/disable the full screen navigation swipe gesture. When this setting is disabled, you can still swipe to go back by swiping from the edge of the screen. This setting has been placed under Settings -> Gestures.

Additionally, I've added all remaining strings to be localized for the Gesture settings page. There were a few duplicate localization strings, so I've removed them here as well.

## Issue Being Fixed

Issue Number: #815

## Screenshots / Recordings

simulator_screenshot_948C3427-A5DD-4495-ACC9-04E0D7B4A33B

## Checklist

- [ ] Did you update CHANGELOG.md?
- [x] Did you use localized strings where applicable?
- [x] Did you add `semanticLabel`s where applicable for accessibility?

### CODE REVIEW

File: lib/l10n/app_en.arb
Comment (micahmo) [inline lib/l10n/app_en.arb:204]: Do these strings need to be added to the other .arb files?

**Review by micahmo — APPROVED** [review]
LGTM!

File: lib/l10n/app_en.arb
Comment (hjiangsu) [reply lib/l10n/app_en.arb:204]: I did a test and it seems like its not needed. I'm also setting up Weblate for translations so it might be better to keep the other files empty! https://hosted.weblate.org/projects/thunder/thunder/

Comment (micahmo) [reply lib/l10n/app_en.arb:204]: Very nice!! Should we go through and remove any English strings in non-English files then?

Comment (hjiangsu) [reply lib/l10n/app_en.arb:204]: Yup, I think we can do that. We can also go ahead and remove any `.arb` files that have not yet been translated. This will just help the Weblate statuses better reflect the overall translation progress.

I've set up hooks so that Weblate automatically pushes new translations to the `translations` branch if you're interested. It seems like we already have a new language translated!

I'm holding off from making an announcement just yet until Weblate approves my application for libre (free) status on the project

**Pull request merged**
